app/errors/handlers.py

```diff
@@ -14,7 +14,19 @@ def page_not_found(e):
     ]
 
     # Set the 404 status explicitly
-    return standardize_response(None, errors, "not found"), 404
+    return standardize_response(None, errors, "not found", 404)
+
+
+@bp.app_errorhandler(429)
```

Comment:
I assume that flask-limiter raises a 429 which is how this handler gets called.

Reply:
Yes, flask-limiter raises a 429 which returns HTML to the client. This method returns JSON instead